docs/azure_logs.qmd

```diff
@@ -0,0 +1,144 @@
+---
```

Comment:
It seems like we can remove this page as we have a more distilled version available in the main eval_logs article?